float32 ambient_celsius
float32 interior_celsius
float32 battery_celsius